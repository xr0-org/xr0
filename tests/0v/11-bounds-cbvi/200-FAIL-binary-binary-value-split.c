#ifdef XR0

void
foo(int i, int j) ~ [
        setup: {
                i = [0?2];
                j = [0?2];
        }
];

#endif

void
foo(int i, int j)
{
	int k, arr[2];
        arr[0] = 0;
        arr[i] = 2;
        arr[j] = 3;
        k = arr[0];
        k = arr[1];
}
