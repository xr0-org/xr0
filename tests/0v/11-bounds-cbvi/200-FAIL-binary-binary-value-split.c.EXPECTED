11-bounds-cbvi/200-FAIL-binary-binary-value-split.x:21:19: undefined memory access: *(arr+1) has no value (foo | #0 ∈ {0} | #1 ∈ {0})
